independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json

```diff
@@ -0,0 +1,514 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "De Lijn Zoek Open Data Services (v1)",
```

Comment:
@Lschockaert the title exceeds the limit of 30 characters. Please see our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)

Reply:
Should I add '(Independent Publisher)' to the title? I checked some examples in the github or is 'De Lijn' ok?